docs/integration/solrcloud.md

```diff
@@ -1,9 +1,112 @@
 title: SolrCloud Monitoring Integration
 description:  Our monitoring and logging platform includes integration for SolrCloud. Use predefined key metrics reports combined with rich data visualization tools to monitor critical issues on your Solr machines cluster, and receive alerts on memory usage, uptime, load averages, index stats, document and filter caches, latency, rate, and more
 
+Sematext offers simple and versatile SolrCloud monitoring agent written in Java and Golang without CPU and memory overhead on your application. It's easy to install and require no changes in SolrCloud source code or your application source code. 
```

Comment:
Sematext offers simple and versatile SolrCloud monitoring agent written in Java and Golang without CPU and memory overhead *. It's easy to install and **requires** * no changes in **the** * SolrCloud source code or your **application's** * source code.

Reply:
Adjusted.